Formula/neovim.rb

```diff
@@ -0,0 +1,60 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io/"
+  url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+  sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+  head "https://github.com/neovim/neovim.git"
+
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "jemalloc"
+  depends_on "libtermkey"
+  depends_on "libuv"
+  depends_on "libvterm"
+  depends_on "lua@5.1"
+  depends_on "luajit"
+  depends_on "msgpack"
+  depends_on "unibilium"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
```

Comment:
If you did see explicit Python framework links in the linkage output, it would actually be a "bug" as we prefer `-undefined dynamic_lookup` for Python.

>So is the logic that anything post snow leopard will build with python automatically since its shipped with macOS ?

Yes, I assume so, though you'd have to check the CMake output to find out if the Python stuff is getting built or not currently.

(Note Snow Leopard also has Python, but it's just too old.)